django/contrib/admin/views/autocomplete.py

```diff
@@ -24,12 +25,19 @@ def get(self, request, *args, **kwargs):
             return JsonResponse({'error': '403 Forbidden'}, status=403)
 
         self.term = request.GET.get('term', '')
-        self.paginator_class = self.model_admin.paginator
```

Comment:
This should be removed in a separate commit if it's unused.

Reply:
haha, can't sneak anything past you. Very good point ;)